recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "6.1.0" %}
+{% set version = "6.2.0" %}
 
 package:
   name: nbconvert
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/n/nbconvert/nbconvert-{{ version }}.tar.gz
-  sha256: d22a8ff202644d31db254d24d52c3a96c82156623fcd7c7f987bba2612303ec9
+  sha256: 16ceecd0afaa8fd26c245fa32e2c52066c02f13aa73387fffafd84750baea863
 
 build:
   number: 0
```

Comment:
Please update entry_points to:
```
  entry_points:
    - jupyter-nbconvert = nbconvert.nbconvertapp:main
    - jupyter-dejavu = nbconvert.nbconvertapp:dejavu_main
```

see https://github.com/jupyter/nbconvert/blob/cefe0bfe303e5e9e194c393cb9280c64a77b8219/setup.py#L259

Reply:
Updated the entry points:
https://github.com/AnacondaRecipes/nbconvert-feedstock/blob/update_to_6.2.0/recipe/meta.yaml#L15&L16